### ISSUE TITLE
Release TwistedWords-0.5.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2362]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#2362|
|Type|enhancement|
|Created|2007-01-06 21:52:48Z|

<details><summary>Searchable metadata</summary>

```
trac-id__2362 2362
type__enhancement enhancement
reporter__radix radix
priority__normal normal
milestone__Words_0_5 Words-0.5
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1168120368000000 1168120368000000
changetime__1168135518000000 1168135518000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_1"></a>|@radix set owner to @ralphm|
|-|-|

Ready for review in releases/twisted-words-0.5.x-2362

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @radix|
|-|-|

There's a tab in NEWS.

I guess we don't really care that a bunch of the entries don't have ticket numbers?  We should make sure to do better next time though.

Otherwise good.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_3"></a>|@radix set status to `closed`|
|-|-|

(In [19232]) Merge twisted-words-0.5.x-2362

Author: release automation, radix
Reviewer: exarkun
Fixes [#2362](https://github.com/twisted/twisted/issues/2362)

Version numbers and NEWS from words 0.5.0

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
